js/client/modules/@arangodb/testsuites/resilience.js

```diff
@@ -89,15 +91,15 @@ var _resilience = function(path) {
   };
 };
 
-const resilienceMove = (new _resilience('resilience_move')).func;
-const resilienceMoveView = (new _resilience('resilience_move_view')).func;
-const resilienceRepair = (new _resilience('resilience_repair')).func;
-const resilienceFailover = (new _resilience('resilience_failover')).func;
-const resilienceFailoverFailure = (new _resilience('resilience_failover_failure')).func;
-const resilienceFailoverView = (new _resilience('resilience_failover_view')).func;
-const resilienceTransactions = (new _resilience('resilience_transactions')).func;
-const resilienceSharddist = (new _resilience('resilience_sharddist')).func;
-const resilienceAnalyzers = (new _resilience('resilience_analyzers')).func;
+const resilienceMove = (new _resilience('resilience_move', false)).func;
+const resilienceMoveView = (new _resilience('resilience_move_view', true)).func;
+const resilienceRepair = (new _resilience('resilience_repair', true)).func;
+const resilienceFailover = (new _resilience('resilience_failover', false)).func;
+const resilienceFailoverFailure = (new _resilience('resilience_failover_failure', false)).func;
+const resilienceFailoverView = (new _resilience('resilience_failover_view', false)).func;
+const resilienceTransactions = (new _resilience('resilience_transactions', false)).func;
+const resilienceSharddist = (new _resilience('resilience_sharddist', true)).func;
+const resilienceAnalyzers = (new _resilience('resilience_analyzers', true)).func;
```

Comment:
Why `AliveMonitor` is disabled for some of them?

Reply:
These tests SIG_STOP arangods from outside of the `instanceManager`, we can't handle that.